SpatialGDK/Source/SpatialGDKServices/Public/SpatialGDKServicesModule.h

```diff
@@ -23,6 +23,8 @@ class SPATIALGDKSERVICES_API FSpatialGDKServicesModule : public IModuleInterface
 	static FString GetSpatialOSDirectory(const FString& AppendPath = TEXT(""));
 	static FString GetSpatialGDKPluginDirectory(const FString& AppendPath = TEXT(""));
 
+	static void DeleteSchemaDatabase();
```

Comment:
Any reason for this to be here and not with the other schema related stuff?

Reply:
Re comment about using TryLoadExistingSchemaDatabase() in DeleteSchemaDatabase(): TryLoadExistingSchemaDatabase() is tailored for actually generating schema - error messages mention generate schema, ClearGeneratedSchema() is sometimes called

Reply:
good point, moving to SpatialGDKEditor